scripts/test/nightly/runTests.sh

```diff
@@ -69,27 +67,56 @@ set +e
 
 END_EXIT_CODE=0
 
+docker buildx rm sleeper
+docker buildx create --name sleeper --use
```

Comment:
It looks like these commands should probably be before the `set +e`?

Following up on my previous comment: https://github.com/gchq/sleeper/pull/5762#discussion_r2513803014

Reply:
I just realised this is only nearly true. The rm command needs to be able to fail, in case the builder already exists. If the create command fails, it should probably fail everything.